takwimu/views.py

```diff
@@ -133,3 +135,22 @@ def form_invalid(self, form):
         print('\n\n\n\n\n\n')
         print form.data
         return super(SupportServicesIndexView, self).form_invalid(form)
+
+# view for testing search functionality
+def search_view(request):
+    # TODO: 13/08/2018 Remove view, url config and template
```

Comment:
Just some thoughts:

 + Why not CBV or Generic CBV?
 + Shouldn't we use `GET` for displaying search page and `POST` to do the actual search?

Reply:
@kilemensi here I was doing the easiest thing to get the search page working for testing, but I'll refactor and use CBV

Reply:
@kilemensi I don't think we need to do POST for the search because the post query is not sensitive. I thought POST was for sending secure data?

Reply:
Yeah, we don't *need* `GET` in this case (especially since we want the searches to be bookmark-able also), but `POST` is for anything that _may change_ things on the server i.e. has side effects.

Reply:
@kilemensi still a little bit confused here, so do I use POST for the search meaning we'll lose the bookmark-ableness of the search URL or I keep the GET?